source/transactions/tests/delete.json

```diff
@@ -195,13 +194,16 @@
         },
         {
           "name": "deleteOne",
+          "object": "collection",
+          "collectionOptions": {
+            "writeConcern": {
+              "w": "majority"
+            }
+          },
           "arguments": {
             "filter": {
               "_id": 1
             },
-            "writeConcern": {
-              "w": "majority"
-            },
             "session": "session0"
```

Comment:
Not that order matters, but in the actual methods "session" is usually the first argument.

Reply:
Easy enough. Done.